### PR TITLE
Package opam-file-format.2.1.1

### PR DESCRIPTION
**Pull request by rjbou**
### `opam-file-format.2.1.1`
Parser and printer for the opam file syntax

---
* Homepage: https://opam.ocaml.org
* Source repo: git+https://github.com/ocaml/opam-file-format
* Bug tracker: https://github.com/ocaml/opam-file-format/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

**Comment by rjbou** [comment]
This release deprecates (almost) all functions. Should I add `<2.1.1` constraint to all packages that depends on ?

**Comment by kit-ty-kate** [comment]
> This release deprecates (almost) all functions. Should I add `<2.1.1` constraint to all packages that depends on ?

None of the revdeps seem to fail. Deprecation is fine unless one of the revdeps use `-warn-error` in releases, which on top of being bad packaging would make me quite angry.
Thankfully none of the revdeps are in this case ^_^

File: packages/opam-file-format/opam-file-format.2.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "alcotest" {with-test}
+]
+depopts: [
+  "dune" {> "1.11.4"}
+]
+build: [
+  [make "byte" {!ocaml-native} "all" {ocaml-native}] {!dune:installed}
+  ["dune" "build" "-p" name "-j" jobs "@install" "@doc" {with-doc}]
+    {dune:installed}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test & dune:installed}
```
Comment (rjbou) [inline packages/opam-file-format/opam-file-format.2.1.1/opam:19]:
```suggestion
  [make "byte" {!ocaml-native} "all" {ocaml-native}]
    {!dune:installed | dune:version < "2.0.0"}
  ["dune" "build" "-p" name "-j" jobs "@install" "@doc" {with-doc}]
    {dune:installed & dune:version > "2.0.0"}
  ["dune" "runtest" "-p" name "-j" jobs] {with-test & dune:installed & dune:version > "2.0.0"}

```

File: packages/opam-file-format/opam-file-format.2.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://opam.ocaml.org"
+bug-reports: "https://github.com/ocaml/opam-file-format/issues"
+depends: [
+  "ocaml"
+  "alcotest" {with-test}
+]
+depopts: [
+  "dune" {> "1.11.4"}
+]
```
Comment (rjbou) [inline packages/opam-file-format/opam-file-format.2.1.1/opam:14]:
```suggestion
depopts: [
  "dune" {> "2.0.0"}
]
```

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by rjbou** [comment]
Thanks!

**Pull request merged**